docs/modules/plugins/pages/plugin-rest-api.adoc

```diff
@@ -36,6 +36,13 @@ a|`true`
 |`false`
 |Enable logging of HTTP request/response headers and bodies (applied to the following content types only: `text/*`, `application/json`, `application/xml`)
 
+|`rest-api.http.cookie-store-level`
+a|`global`
+`story`
+`scenario`
+|`global`
+|The property that allows to configure level of storing cookies during execution requests.
```

Comment:
please provide more information for each level value.

Reply:
updated, thanks